app/main/views/marketplace.py

```diff
@@ -83,6 +83,13 @@ def index():
     )
 
 
+@main.route('/_test_harakiri', methods=['GET'])
+def test_timeout():
+    from time import sleep
+    sleep(300)
+    abort(500)
```

Comment:
I'd probably put this in `app.status.views.py` rather than `marketplace.py`

Reply:
We can not unit test, maybe e2e test, but even then there is no way around having to wait more than 60 seconds (nginx default timeout and our harakiri value).